xcdat/spatial.py

```diff
@@ -281,12 +281,6 @@ def get_weights(
                     "to reference a specific data variable's axis bounds."
                 )
 
-            # The logic for generating longitude weights depends on the
-            # bounds being ordered such that d_bounds[:, 0] < d_bounds[:, 1].
-            # They are re-ordered (if need be) for the purpose of creating
-            # weights.
-            d_bounds = self._force_domain_order_low_to_high(d_bounds)
-
```

Comment:
This is not necessary for latitude because we use the [absolute difference of the domain bounds](https://github.com/xCDAT/xcdat/blob/main/xcdat/spatial.py#L515). So the logic was updated in the `_get_longitude_weights()` function.

Reply:
Note that this also caused the error

> xarray can't set arrays with multiple array indices to dask yet.

because `d_bounds` were not loaded / copied.